model/inline/src/main/scala/aqua/model/inline/TagInliner.scala

```diff
@@ -229,6 +235,24 @@ object TagInliner extends Logging {
             )
         }
 
+      case PushToMapTag(key, operand, exportTo) =>
+        (
+          valueToModel(key),
+          valueToModel(operand),
+          valueToModel(exportTo.toRaw)
+        ).mapN {
+          case ((k, kp), (v, p), (VarModel(name, st, Chain.nil), None)) =>
```

Comment:
nit: Can we annotate `st` with `StreamMapType` here just to check that tag is well-formed?